custom_components/localtuya/light.py

```diff
@@ -366,10 +370,10 @@ def status_updated(self):
         self._state = self.dps(self._dp_id)
         supported = self.supported_features
         self._effect = None
-        if supported & SUPPORT_BRIGHTNESS:
+        if supported & SUPPORT_BRIGHTNESS and self.has_config(CONF_BRIGHTNESS):
             self._brightness = self.dps_conf(CONF_BRIGHTNESS)
 
-        if supported & SUPPORT_COLOR:
+        if supported & SUPPORT_COLOR and self.has_config(CONF_COLOR):
```

Comment:
Same as above.

Reply:
Removed overzealous check. Don't know why I did that